internal/tools/aws/constants.go

```diff
@@ -203,6 +203,15 @@ const (
 	// schemas allowed in a Posgres multitenant RDS database cluster.
 	DefaultRDSMultitenantDatabasePostgresCountLimit = 300
 
+	// DefaultRDSMultitenantPGBouncerDatabasePostgresCountLimit is the maximum
+	// number of schemas allowed in a Posgres multitenant RDS database cluster
```

Comment:
```suggestion
	// number of schemas allowed in a Postgres multitenant RDS database cluster
```

Reply:
Thanks! I will clean this up in a change I am working on now.